src/main.cpp

```diff
@@ -804,10 +810,12 @@ int main ( int argc, char** argv )
     // init resources
     Q_INIT_RESOURCE ( resources );
 
+#ifndef SERVER_ONLY
     // clang-format off
 // TEST -> activate the following line to activate the test bench,
 //CTestbench Testbench ( "127.0.0.1", DEFAULT_PORT_NUMBER );
     // clang-format on
```

Comment:
Apparently line 817 shouldn't have leading spaces.  Presumably because it's confused by line 818?

Reply:
If anyone has a clue why `make clang_format` and the clang format check disagree, that would help, too.

Reply:
Let's follow up on this in https://github.com/jamulussoftware/jamulus/issues/2552#issuecomment-1080020627.